src/benchmarks/micro/libraries/Microsoft.Extensions.Caching.Memory/MemoryCacheTests.cs

```diff
@@ -22,7 +26,7 @@ public void Setup()
             }
         }
 
-        [GlobalCleanup]
+        [GlobalCleanup(Targets = new[] { nameof(GetHit), nameof(TryGetValueHit), nameof(GetMiss), nameof(TryGetValueMiss), nameof(SetOverride) })]
         public void Cleanup() => _memCache.Dispose();
```

Comment:
```suggestion
        public void CleanupBasic() => _memCache.Dispose();
```

To match `SetupBasic`?